models/example/base/my_first_dbt_model.sql

```diff
@@ -6,5 +6,7 @@
 -- Try changing 'view' to 'table', then re-running dbt
 {{ config(materialized='view') }}
 
+--it is best practice to only reference raw data in base models
+--https://dbt.readme.io/docs/best-practices#section-limit-dependencies-on-raw-data
```

Comment:
use this link instead:

https://docs.getdbt.com/docs/best-practices#section-limit-dependencies-on-raw-data